hw20/hosts.yml

```diff
@@ -0,0 +1,14 @@
+#Server list
+
+all:
+ children:
+  linux:
+   hosts:
+    192.168.1.200:
+    192.168.1.201:
+  ubuntu:
+   hosts:
+    192.168.1.200
```

Comment:
Use hosts Its only once. Use group name instead of host IP

Reply:
For example
```
all:
  children:
    testhost1:
      hosts:
        172.30.128.3
    testhost2:
      hosts:
        172.30.128.4
    centos:
      hosts:
        172.30.128.5
    mix:
      children:
        testhost1:
        centos:

Reply:
fix